test-infra/auto-deploy/deploy-master.yaml

```diff
@@ -22,14 +22,16 @@ spec:
           value: /secret/gcp-credentials/key.json
         command:
         - /usr/local/bin/auto_deploy.sh
-        - --repos=kubeflow/kubeflow;kubeflow/testing
+        #- --repos=kubeflow/kubeflow;kubeflow/testing
+        - --repos=kubeflow/kubeflow;jlewi/testing@fix_nfs2
```

Comment:
Do we need to change this back to `kubeflow/testing` after merging this pr?

Reply:
Done.